src/TemplateData.js

```diff
@@ -104,7 +120,7 @@ class TemplateData {
 
   async _checkInputDir() {
     if (this.inputDirNeedsCheck) {
-      let globalPathStat = await fs.stat(this.inputDir);
```

Comment:
We chatted a little over DM about this—are there any good resources to learn the appropriate use of async versus sync? I usually favor async for most things

Reply:
Honestly, given that I don't think there's gonna be much of a difference here for quick SSDs (it would likely boil down to task ordering rather than I/O itself). Will back this change out.